src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -298,7 +297,24 @@ public final void setAlpha(float opacity)
         super.setAlpha(opacity);
     }
 
-    public boolean dispatchTouchEvent(MotionEvent e)
+	/*
+	// To trace the 'dispatchTouchEvent', uncomment this and then uncomment logs in the method itself
+	private static String _indent = "";
+	public boolean dispatchTouchEvent(MotionEvent e)
+	{
+		String originalIndent = _indent;
+		Log.i("", _indent + "    + " + this.toString());
```

Comment:
Is this costly?

Reply:
Oh yeah! (When dragging a `Slider` for instance)